install_magenta.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Check which operating system
+if [ "$(uname)" == "Darwin" ]; then
+    echo "Mac OS Detected"
+    export OS="MAC"
+    export MINICONDA_SCRIPT=Miniconda2-latest-MacOSX-x86_64.sh
+    # Mac OS X, CPU only, Python 2.7:
+    export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-0.11.0rc0-py2-none-any.whl
+elif [ "$(uname)" == "Linux" ]; then
```

Comment:
I'm not sure the best way to do this, but we probably only want to proceed if they're running a Debian variant since we expect to call apt-get later in the script. I think you could check for the presence of /etc/debian_version, or maybe just see if apt-get is available on the path.